### DISCUSSION TITLE
Stitching images with transparency

### DISCUSSION THREAD

**Discussion by alex-rothwell** [OpenStitching/stitching#212] [Q&A]
I have two images which I want to stitch together. However, the part of the image that I am interested in is a circle in the centre, that is surrounded by a black border, as shown below.

<img src="https://github.com/OpenStitching/stitching/assets/41335141/a75e7cbf-0c1a-4054-a160-d6332f1a6589" width="200">
<img src="https://github.com/OpenStitching/stitching/assets/41335141/c61700c2-18b6-4467-a178-1eac61778851" width="200">

Therefore, I've cropped the image to remove the black background to generate the below images.

<img src="https://github.com/OpenStitching/stitching/assets/41335141/8d06c65b-7d33-49a2-9b8e-a5d2ea3099bc" width="200">
<img src="https://github.com/OpenStitching/stitching/assets/41335141/41ce2755-1c5d-424a-afe3-6782b6ddce5a" width="200">

As the area that is cropped from the image is now transparent, to read the image in using opencv requires the `cv.IMREAD_UNCHANGED `flag ie. `cv.imread("images/preprocess/crop_img1.png", cv.IMREAD_UNCHANGED)`

This works fine when reading in each individual image, but when running `stitcher.stitch_verbose ` I get the following error:

```
panorama = stitcher.stitch_verbose([cv.imread("images/preprocess/crop_img1.png", cv.IMREAD_UNCHANGED),
                                    cv.imread("images/preprocess/crop_img2.png", cv.IMREAD_UNCHANGED)])
```

`error: OpenCV(4.9.0) D:\a\opencv-python\opencv-python\opencv\modules\stitching\src\timelapsers.cpp:74: error: (-215:Assertion failed) img.type() == CV_16SC3 in function 'cv::detail::Timelapser::process'`

Any ideas or workarounds? Thanks

**Comment by lukasalexanderweber** [comment]
You can just set `--timelapse no` or Stitcher `stitcher = Stitcher(timelapse="no")`

**Reply by alex-rothwell** [reply]
Thanks for the response Lukas. I'm getting the error with timelapse="no"
